ITE/6/spec/sbom.md

```diff
@@ -0,0 +1,48 @@
+# Attestation type: SBOM v1
+
+## Purpose
+
+A Software Bill of Materials type following the SPDX 3.0/3T SBOM std.
+
+This allows to represent an "exportable" or "published" software artifact. It
+can also be used as an entry point for other types of in-toto attestation when
+performing policy decisions
+
+## Schema
+
+```jsonc
+{
+  "attestation_type": "https://in-toto.io/Provenance/v1",
```

Comment:
Should 'attestation_type' say something about SBOM instead of 'Provenance' ?

Reply:
Fixed. It is now SPDX, which is the specific format. (SBoM is not a specific format.)